README.md

```diff
@@ -4,14 +4,67 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
 
+## Installation
 `gem install webmock-fixtures`
 
 or with Bundler
 
 `gem 'webmock-fixtures`
 
+## Getting Started
+WebMock fixtures provides a way to pre-define all of your fixtures ahead of time and start them easily during tests.
+
+```ruby
+require "httparty"
+require "webmock/fixtures"
+
+# Register a fixture as `:get_example`
+WebMock::Fixtures::Manager.register_fixture(
+  :get_example, :get, %r{www.example.org}, :body => "Hello World")
+
+# Create a manager with the `:get_example` fixture started
+manager = WebMock::Fixtures::Manager.run([:get_example])
+
+# Make our HTTP request
+response = HTTParty.get("http://www.example.org")
+
+# Assert we received our mock response
+if response.body != "Hello World"
+  fail("Response body \"#{response.body}\" was not what we expected")
+end
+
+# Access the created `WebMock::RequestStub` for our `:get_example` fixture
+stub = manager[:get_example]
+```
+
+As well, by subclassing `WebMock::Fixtures::Manager` you can easily separate and organize your test fixtures. For example:
+
+```ruby
+# Manager to store fixtures associated with Google
+class GoogleManager < WebMock::Fixtures::Manager
+  # Define fixtures in class definition
+  @registered_fixtures = {
```

Comment:
I mean on `initialize` you could process all `@fixtures` into `@registered_fixtures` via the `register_fixture` mechanism. This would guarantee everything is added consistently and `@fixtures` is the proper format (e.g. a Hash)